views/map.js

```diff
@@ -275,7 +277,10 @@ function featureCollection (features) {
 
 function getBounds (features = []) {
   const extent = [[-180, -85], [180, 85]]
-  for (const { geometry: { coordinates: [lon, lat] = [] } = {} } of features) {
+  //Filters all features without a coordinate
+  const filteredFeatures = features.filter(feat => (typeof feat.geometry.coordinates !== 'boolean' && feat.geometry.coordinates[0] !== null && feat.geometry.coordinates[1] !== null))
```

Comment:
think it would be sufficient (and more resilient) to check that `geometry.coordinates` is an array of length 2 with non-null elements e.g. 

```js
Array.isArray(feat.geometry.coordinates) && 
feat.geometry.coordinates.length === 2 && // can probably omit this check if desired
feat.geometry.coordinates[0] !== null &&
feat.geometry.coordinates[1] !== null)
```